wp-content/themes/csisnuclearnetwork/assets/_scss/components/_footnotes.scss

```diff
@@ -38,47 +42,59 @@
 
   &__list {
     padding-left: 0;
-    font-size: rem(16);
-    color: $color-black-190;
-    @extend %text-style-paragraph-medium-short;
   }
 }
 
 .easy-footnotes-wrapper {
   @extend %text-style-paragraph-medium-short;
   padding-left: 0;
+  line-height: 1.3;
   counter-reset: item;
 
   li {
     position: relative;
     left: 8px;
     display: block;
-    color: $color-black-190;
+    padding-left: rem(16) !important;
+    color: $color-black-180;
+
+    &::after {
+      position: absolute;
+      top: 0;
+      left: -8px;
+      display: block;
+      margin-left: 0;
+      content: counter(item) '.';
+      @extend %text-style-paragraph-medium-short;
+      color: $color-black-180;
+      counter-increment: item;
+    }
 
     &:not(:last-child) {
       margin-bottom: rem(12);
     }
 
+    a:not(.easy-footnote-to-top) {
+      color: $color-accent-primary-blue-200;
```

Comment:
We can remove the color from this, as it will be handled with the post content styles.